### ISSUE TITLE
CITGM shows "undefined" failed jobs

### ISSUE THREAD

**Issue by BridgeAR** [nodejs/build#1444]
I am not certain if this is a recent CITGM issue or if it's build related, while I expect it to be the latter.

See https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/1505/#showFailuresLink

**Comment by maclover7** [comment]
The undefined versions only show up on AIX machines (cc @nodejs/platform-aix) -- had been meaning to open up an issue about this

**Comment by gdams** [comment]
it's this `EACCES: permission denied, mkdir '/ramdisk0/citgm'`

**Comment by gdams** [comment]
I've run `chown -R iojs /ramdisk0/` on `test-osuosl-aix61-ppc64_be-2`. Let's see if that fixes the issue

**Comment by refack** [comment]
> it's this EACCES: permission denied, mkdir '/ramdisk0/citgm'

This rings a bell... 
https://github.com/nodejs/build/issues/895#issuecomment-331650448
tl;dr to clean, run `rm -Rf /ramdisk0/citgm/*` otherwise there might be ownership issues.
